skrub/_column_associations.py

```diff
@@ -18,14 +18,14 @@ def column_associations(df):
 
     The result is returned as a dataframe with columns:
 
-    ['left_column_name', 'left_column_idx', 'right_column_name',
-    'right_column_idx', 'cramer_v']
+    `['left_column_name', 'left_column_idx', 'right_column_name',
+    'right_column_idx', 'cramer_v']`
```

Comment:
```suggestion
    ``['left_column_name', 'left_column_idx', 'right_column_name',
    'right_column_idx', 'cramer_v']``
```

Reply:
sphinx uses restructured text which unlike markdown wants 2 backticks for formatting code :shrug: one backtic gives italic font -- I don't think I well ever get used to it